### PR TITLE
Bb

### PR DESCRIPTION
**Pull request by brendan6780**
Dunno why it won't automaticall merge?

### CODE REVIEW

File: src/testjfxapp/Scoreboard.java [hunk 1]
```diff
@@ -14,6 +20,8 @@
     Button btm1;
 
     MainMenu mainMenu;
+    Tetris tetris = new Tetris(10, 20, 30, mainMenu);
+    // Add same for tetsaw class
```
Comment (BlehCJHunter) [inline src/testjfxapp/Scoreboard.java:24]: Do I need to explain again why this is not what you should be doing?
Add a Tetris object to your constructor's inputs, and assign the variable to that. You should only ever make a new Tetris object when you are starting a new game.

File: src/testjfxapp/Scoreboard.java [hunk 2]
```diff
@@ -26,13 +34,14 @@ public void start(Stage primaryStage) throws Exception {
 
         btm1 = new Button("Back to Menu");
         btm1.setOnAction(e -> mainMenu.showMenu());
+        Label tetrisScore = new Label("Your last score in Tetris was "+this.calculateTetrisScore());
+        Label tetsawScore = new Label("Your last score in Tetsaw was "+this.calculateTetrisScore()); // Change to tetsaw when implemented
```
Comment (BlehCJHunter) [inline src/testjfxapp/Scoreboard.java:38]: Unneeded `this.`

File: src/testjfxapp/Tetris.java [hunk 3]
```diff
@@ -116,6 +116,8 @@ public void start(Stage arg0) throws Exception {
                     break;
                 case ESCAPE: try {
                     this.pause();
+                    this.getBlockScore();
+                    this.getTickScore();
```
Comment (BlehCJHunter) [inline src/testjfxapp/Tetris.java:129]: Unneeded `this.`

**Review by BlehCJHunter — CHANGES_REQUESTED** [review]

File: src/testjfxapp/Scoreboard.java (hunk 2 above)
Comment (brendan6780) [reply src/testjfxapp/Scoreboard.java:38]: I need all the this's

**Pull request merged**